libraries/radpdfprocessing/model/general-information.md

```diff
@@ -24,6 +21,13 @@ This article explains the structure of __RadPdfProcessing__'s document model.
 __RadFixedDocument__ represents a tree of [RadFixedPage]({%slug radpdfprocessing-model-radfixedpage%}) where the fixed content is hosted. The diagram below describes the composition of the fixed content. The document elements are denoted in black and collections - in orange.
         ![Rad Pdf Processing Model 02](images/RadPdfProcessing_Model_02.png)
 
+## Creating or Editing document content 
```

Comment:
## Creating or Editing Document Content 

The RadPdfProcessing library provides API for editing existing or creating brand new documents. This is achieved via the following editors: 
* [RadFixedDocumentEditor]({%slug radpdfprocessing-editing-radfixeddocumenteditor%}): This editor is suitable for creating new documents or adding content to existing documents. It allows you to add elements in a flow-like manner without explicitly setting positions and sizes. The RadFixedDocumentEditor takes care to arrange the document elements automatically and separate the content on different pages when needed.

* [FixedContentEditor]({%slug radpdfprocessing-editing-fixedcontenteditor%}): This editor provides you with the great flexibility of the PDF format. It is suitable for adding content to existing pages. With it, you must specify the exact position of each new element and manually separate the content on different pages when needed.